src/components/ListItem.jsx

```diff
@@ -1,28 +1,44 @@
 import './ListItem.css';
 import { updateItem, deleteItem } from '../api/firebase.js';
-import { useState, useEffect, useMemo } from 'react';
+import { useState, useEffect } from 'react';
 import { ONE_DAY_IN_MILLISECONDS } from '../utils/dates.js';
 import { purchaseUrgency } from '../utils/hooks.js';
 
 export function ListItem({ listPath, item }) {
-	/* Returns a boolean that is passed into isChecked useState
-	On render, box is checked if purchased less than a day ago */
+	//Time variables for each item on render
+	const lastPurchase =
+		item.dateLastPurchased[item.dateLastPurchased.length - 1];
+	const timeElapsed = Date.now() - lastPurchase.seconds * 1000;
 
-	const purchasedOneDayAgo = useMemo(() => {
-		if (item.dateLastPurchased === null) {
+	//Boolean to pass into isChecked state
+	const purchasedWithinDay = () => {
+		//Exclude new, never purchased items - new items are added to db with same dateCreated + first dateLastPurchased
+		if (item.dateCreated.seconds === lastPurchase.seconds) {
```

Comment:
praise: Good catch on the never purchased items!

Reply:
praise: Agreed! I really appreciate you considering the actual use of the the application and how it would throw off our users to have unexpected behavior like this.